ludwig/schema/features/preprocessing/category.py

```diff
@@ -46,13 +46,24 @@ class CategoryPreprocessingConfig(BasePreprocessingConfig):
     )
 
     most_common: int = schema_utils.PositiveInteger(
-        default=10000,
+        default=None,
         allow_none=True,
         description="The maximum number of most common tokens to be considered. if the data contains more than this "
         "amount, the most infrequent tokens will be treated as unknown.",
         parameter_metadata=FEATURE_METADATA[CATEGORY][PREPROCESSING]["most_common"],
     )
 
+    most_common_percentile: int = schema_utils.FloatRange(
+        default=1,
```

Comment:
Should this be 0.95?

Reply:
My rationale is that since the vocab size of category, bag, set features is small, I set these to 1.

Reply:
Updated